packages/postcss-normalize-url/src/index.js

```diff
@@ -46,6 +46,11 @@ function transformNamespace(rule) {
 }
 
 function transformDecl(decl, opts) {
+  // Ref https://github.com/cssnano/cssnano/issues/927
+  if (typeof decl.value !== 'string') {
+    decl.value = decl.value.toString();
+  }
```

Comment:
`decl.value` should be always `string`

Reply:
but it is coming as non string

```js
  nodes: [
        { type: 'word', sourceIndex: 0, value: '2' },
        { type: 'space', sourceIndex: 1, value: ' ' },
        { type: 'word', sourceIndex: 2, value: 'auto' }
      ]
```

Reply:
We need investigate why we have array there and should fix it where we put array this, not here